packages/devui-vue/devui/tree/src/tree.tsx

```diff
@@ -18,31 +28,43 @@ export default defineComponent({
   setup(props: TreeProps, context: SetupContext) {
     const { slots, expose } = context;
     const treeInstance = getCurrentInstance();
-    const { data, check, operate } = toRefs(props);
+    const { check, draggable, operate } = toRefs(props);
```

Comment:
`draggable`会理解为是一个boolean值，建议叫`drag`，代表这是节点拖拽功能
```suggestion
    const { check, drag, operate } = toRefs(props);
```